src/api/app/views/webui2/webui/project/list.html.haml

```diff
@@ -0,0 +1,51 @@
+:ruby
+  @pagetitle = 'Public Projects'
+.card
+  - if @important_projects.present?
+    .card-body
+      %h3 Main Projects
+      .table-responsive
+        %table.table.table-sm.table-striped.table-bordered
```

Comment:
I'm not sure if this table needs the search/pagination/information... I would add it. What do you think?

Reply:
It was a simple paragraph before, I thought it was better to have a list or even a table. I choosed a table because its content is quite similar to the table at the bottom. I also supposed it would only contain a few lines, so I kept it simple... but ... you make me doubt now
 :thinking: